src/pages/Markdown/index.js

```diff
@@ -29,8 +41,14 @@ class MarkdownPage extends PureComponent {
   }
 
   render() {
+    let helmet;
+    if (this.noindexPages.includes(this.props.page)) {
+      helmet = MarkdownPage.addNoindexMeta();
+    }
+
     return (
       <ContentPageWrapper>
+        {helmet}
```

Comment:
The typical React way to write this would be …

```suggestion
        {this.noindexPages.includes(this.props.page) && <Helmet>
          <meta name="robots" content="noindex" />
        </Helmet>}
```

Which is a bit noisy and merging conditions and templates, but on the other hand it saves quite a bit of noise in the rest of the file.

A bit more here https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator

In this instance, the noise inside the template is IMO preferable to the noise in the class, since we keep the footprint of the feature contained to two places, the `noindexPages` const and the template. (Compared to 4 places right now.)

Reply:
Oh cool, thank's for that input! Didn't know that. I will change it accordingly.